### ISSUE TITLE
SelectButton: AriaLabels on ToggleButtons not set

### ISSUE THREAD

**Issue by andrebernardini** [primefaces/primeng#19616]
### Describe the bug

The individual toggle-buttons don't have the `ariaLabel` set to the `label` of the underlying option as described in the documentation of the SelectButton (in the Accessibility chapter). This poses major issues for users of screenreaders, because they will not be able to identify the buttons and understand what they do.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/nasrbxeg?file=src%2Fapp%2Fselectbutton-template-demo.ts

### Environment

Not relevant (Stackblitz provided).

### Angular version

21

### PrimeNG version

v21

### Steps to reproduce the behavior

1. Configure a select-button with options that have a label set.
2. Inspect the one of the toggle-buttons in the accessibility view of your browser's dev-tools.

### Expected behavior

The ariaLabel on the toggle-button should be set (but is not).
